global_config.yaml.tpl

```diff
@@ -11,27 +11,32 @@ sources:
 
```

Comment:
BTW The PR name says `rename include to exclude`, but it should be `rename ignore to exclude` 😄